pkgs/desktops/gnome-3/3.20/core/gnome-shell/default.nix

```diff
@@ -40,6 +42,9 @@ stdenv.mkDerivation rec {
       --prefix XDG_DATA_DIRS : "${gnome_themes_standard}/share:$out/share:$XDG_ICON_DIRS" \
       --suffix XDG_DATA_DIRS : "$GSETTINGS_SCHEMAS_PATH"
 
+    wrapProgram "$out/bin/gnome-shell-extension-tool" \
+      --prefix PYTHONPATH : "${python3Packages.pygobject3}/${python3.sitePackages}:$PYTHONPATH"
```

Comment:
If I am correct the wrapper won't be needed anymore then, at least, if patchShebangs points to the `python` in the `buildEnv`, and not just the interpreter. Maybe @bennofs knows?